### PR TITLE
Move from Machine images to using AMIs built in Amigo

### PR DESCRIPTION
**Pull request by jennysivapalan**
As we love updated AMIs! I've not added regularly updated AMI with deploy because that requires a change from deploy.json -> riff-raff.yaml. That can be a separate PR.

I've tested this on CODE.

### CODE REVIEW

**Review by clloyd — APPROVED** [review]

**Comment by TBonnin** [comment]
@jennysivapalan you don't need to switch to `riff-raff.yaml`. 
The ami-cloudformation-parameter deployment type in riff-raff can be added to your `deploy.json`

**Comment by jennysivapalan** [comment]
@TBonnin thanks! I didn't realise that.

File: cloudformation/editorial-viewer.json [hunk 1]
```diff
@@ -43,6 +48,10 @@
       "Description": "Github team name, used for giving ssh access to members of the team.",
       "Type": "String",
       "Default": "Editorial-Tools-SSHAccess"
+    },
+    "ImageId": {
```
Comment (TBonnin) [inline cloudformation/editorial-viewer.json]: minor: Default name for the AMI id parameter in Riff-Raff is `AMI`. Calling it this way might save you one line in your `riff-raff.yaml` or `deploy.json`

Comment (jennysivapalan) [reply cloudformation/editorial-viewer.json]: Thanks @TBonnin

**Pull request merged**